frontends/mit-open/src/pages/DepartmentListingPage/DepartmentListingPage.tsx

```diff
@@ -0,0 +1,232 @@
+import React from "react"
+import {
+  Container,
+  Typography,
+  styled,
+  PlainList,
+  List,
+  ListItem,
+  ListItemLink,
+  ListItemText,
+} from "ol-components"
+import type { TypographyProps } from "ol-components"
+import { MetaTags, pluralize } from "ol-utilities"
+import type {
+  LearningResourceSchool,
+  LearningResourceSearchResponse,
+} from "api"
+import {
+  useLearningResourcesSearch,
+  useSchoolsList,
+} from "api/hooks/learningResources"
+import {
+  RiPaletteLine,
+  RiSeedlingLine,
+  RiBriefcaseLine,
+  RiMacbookLine,
+  RiBarChartBoxLine,
+  RiUserSearchLine,
+  RiArrowRightSLine,
+} from "@remixicon/react"
```

Comment:
I decided to start using the remixicon icons now. We should switch all the icons to remixicon since that what Steve and Bilal want to use. There's an issue 

- https://github.com/mitodl/hq/issues/4104

for switching, but there's not really any harm in starting to use them now.